### DISCUSSION TITLE
【能夠閱讀一場籃球比賽的AI】

### DISCUSSION THREAD

**Discussion by Chienyuuu6** [1101-GS4719-C/Python-Community-GS4719-C#1280] [Ideas]
今天早上買咖啡的路上突然想到：系際盃快到了，可是我們人手超不足，根本沒辦法進行場下的數據統計。

我突然想到如果可以訓練AI閱讀一場籃球賽，肯定很猛！會讓一個球隊的運作更有效率，但我覺得困難度蠻高的，因為籃球場上的變化很多，簡單的數據分類就有得分、籃板、抄截、助攻、阻攻、失誤等等的，假設能夠判斷並記錄每個球員的攻守數據真的很強～

如果真的能夠實現的話，我更期待能夠幫助球隊進行攻守模式的分析：例如防守陣型的判斷、漏洞；進攻的跑位還有攻擊效率等等的～

過去還是學生球員時，我們常常要看影片對對手做分析，並記錄很多複雜的數據，常常搞到半夜兩三點，有夠累，而且不只我們球員要做這件事，教練也常常通宵研究對手的各種戰略。雖然很辛苦，但這樣做讓我們在比賽前更能掌握對手，更輕鬆的拿下勝利，我期待未來能夠有這樣的科技！

希望我有能力做出這樣的運動科技AI🤩

**Comment by poweichen00** [comment]
我知道目前有一個叫Second Spectrum，他們使用AI的演算法辨識跟預測對方的打法，也能給予相對應的戰術，每種不同的選擇，推測對應的投籃命中率。

**Reply by Chienyuuu6** [reply]
https://zhuanlan.zhihu.com/p/524196344

我剛剛去查了大概知道你說的東西，但我突然想到，運動心理學在這樣的世代下是不是更顯重要性了呢~

之前遇過教練下達指令要求他的球員不要防守一位平常出手次數很少的控球員，我看到那場比賽直接被帥到，竟然有這樣的策略，但那個控球超猛值接單場四科三分堵住對手的嘴......

如果一位選手的運動表現習慣被AI是破聽起來好像滿驚悚的

**Comment by Chiuhsianglin** [comment]
我覺得如果有這種AI技術的話，對於紀錄台跟球隊都會有很大的幫助

**Comment by PaulSu0905** [comment]
如果有這樣的AI科技出現，是否能夠完全取代裁判的工作，減少人為因素所導致的誤判，進而讓比賽公平性增加，例如之前LeBron James 的誤判事件或許就不會發生，比賽結果也會不同

**Comment by tower20021202** [comment]
如果這樣的AI科技出現，勢必會對賽場上造成極道的影響，許多球員的假摔要犯規等等舉度都將會AI中無所遁形

**Comment by KevinCheng42** [comment]
我覺得要讓AI閱讀球賽，相關硬體設備如攝影機的更新是必要的~
在網路上搜尋AI在籃球賽的相關應用後，我發現雖然沒有作者提到的可以直接閱讀比賽並分析戰術的AI，但不乏有些協助球員練習射籃精準度以及分析特定情況籃下最佳投籃位置的AI，我相信再過不久就會出現能分析籃球賽戰術並提供指導的「AI教練」！

**Comment by Tzuhsuannnnn** [comment]
希望過沒多久之後可以有這樣的AI工具，可以讓賽場上更加公平。學生間的比賽也可以不需要花費人力，或金錢請裁判。

**Comment by CCello** [comment]
我認為在這個領域，AI需要克服的挑戰包括：解讀場上的視覺資訊、掌握比賽中的變化和運作、識別球員和球的位置，以及應對場上的噪音和不確定性等等。不過，現在的AI已經有了很強的圖像識別和模式識別能力，並且在運用大數據和機器學習的技術方面也非常優秀。未來，我們可以期待看到更多關於AI在體育領域的應用，這些應用可能會改變運動比賽的方式和策略，也可能會讓運動員和教練們能夠更好地分析和優化比賽策略。

**Comment by YoYooHuang** [comment]
關於這方面的AI訓練，我覺得還需要一段時間。無論是什麼比賽，規則都因大家能力不斷進步，有許多新技術的出現，也因此帶來了許多規則的改變。而這些新規則的訓練資料又會更少。所以我認為可以AI搭配人工判決，不僅減少誤判，也能保證少見的play被正確的判決！
